src/libs/utils/arrangeByLanguage.ts

```diff
@@ -0,0 +1,13 @@
+import * as R from 'ramda';
+import { SupportedLanguage } from '../i18n';
+import { Contributor } from '../domain/contributor';
+import { Notice } from '../domain/notice';
+
+function arrangeByLanguage<Items extends Notice | Contributor>(
+  items: Items[],
+  locale: SupportedLanguage
+): Items[] {
+  return R.sort(item => (item.locale === locale ? -1 : 1), items);
```

Comment:
Ramda don't bring value here:
```
items.sort(item => (item.locale === locale ? -1 : 1))
```

Reply:
@lutangar I agree, but it was to keep consistency with the rest, I also prefer to change